spring-integration-cassandra/src/test/java/org/springframework/integration/cassandra/test/utils/SICassandraTestUtils.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2015 the original author or authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * @author Filippo Balicchia
+ */
+package org.springframework.integration.cassandra.test.utils;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import org.springframework.integration.cassandra.test.domain.Book;
+
+public class SICassandraTestUtils {
```

Comment:
The `@author` block is in the wrong place here. Should be exactly before class name. (OK before class-level annotations).

From other side I don't like this class name. You do here the `Book` stuff. So, maybe something like `BookFactory` or `BookSampler`, but that definitely isn't `Utils`.
And move it, please, to the `domain` package.